opts/env_test.go

```diff
@@ -5,38 +5,54 @@ import (
 	"os"
 	"runtime"
 	"testing"
+
+	"gotest.tools/v3/assert"
 )
 
 func TestValidateEnv(t *testing.T) {
```

Comment:
I see there's a slightly nicer test in moby/moby; perhaps I'll update the test and copy that version

Reply:
Updated with the tests from upstream moby/moby with the same changes as; https://github.com/moby/moby/pull/41357